Textmining/src/org/wisslab/ss15/textmining/Speaker.java

```diff
@@ -70,22 +70,22 @@ public double getTFIDF(String word) {
         int D = work.getSpeakers().size();
         return tf * Math.log(D/df);
     }
-    
+
     public List<Double> getVSMVector() {
         List<Double> result = new ArrayList<>();
         for (String word: work.getDF().keySet()) {
             result.add(getTFIDF(word));
         }
         return result;
     }
-    
+
     public Map<String, Double> getTF() {
         Map<String, Double> result = new HashMap<>();
         Pattern p1 = Pattern.compile("[a-zA-Z']+");
-     
+
         for (String word: getAllText().split(" ")) {
             // Das ist quasi Tokenization, dafür gibt es Spezialbibliotheken
-            word = word.replaceAll(",", "");
+            word = word.replaceAll("\\,", "");
```

Comment:
Das ist doch das Komma, nicht der Punkt. Muss man Komma in regex auch escapen? Abgesehen davon habe ich den Code ja heute morgen noch mal komplett umgebaut, jetzt muss das wenn schon in NLPWrapper gemergt werden.